core/include/vecmem/memory/details/sycl_builtin_device_atomic_ref.hpp

```diff
@@ -52,9 +49,9 @@ struct builtin_address_space<device_address_space::local> {
 template <typename T,
           device_address_space address = device_address_space::global>
 using builtin_device_atomic_ref =
-    cl::sycl::atomic_ref<T, details::builtin_address_space<address>::ord,
-                         details::builtin_address_space<address>::scp,
-                         details::builtin_address_space<address>::add>;
+    ::sycl::atomic_ref<T, details::builtin_address_space<address>::ord,
```

Comment:
Any particular reason for the anchored namespace? Is it because there's a `vecmem::sycl` namespace?

Reply:
We have `sycl` namespaces all over the place in our code. Beside `vecmem::sycl` we prominently have `traccc::sycl` as well. Without making it explicit that these types come from the "global" `sycl` namespace, we would get all kinds of compilation errors as soon as we tried to use these classes inside one our our own `sycl` namespaces.